src/Components/AllTrainingTable/ConvertDate.jsx

```diff
@@ -0,0 +1,32 @@
+const CovertDate = (date, text) => {
```

Comment:
Move this file into utils folder. It should as a functions will return date needed.